components/Card/index.js

```diff
@@ -26,19 +31,31 @@ const StyledTitleSection = styled.div`
   justify-content: space-between;
 `;
 
+const ProjectImage = styled(Image)``;
+
 export default function Card({ project, favorites, onToggleFavorite }) {
   return (
     <StyledLink key={project._id} href={`projects/${project._id}`}>
       <StyledCard>
-        <StyledTitleSection>
-          <ProjectTitle>{project.title}</ProjectTitle>
-          <FavoriteButton
-            id={project._id}
-            favorites={favorites}
-            onToggleFavorite={onToggleFavorite}
-          ></FavoriteButton>
-        </StyledTitleSection>
-        <ShortFactsBox project={project}></ShortFactsBox>
+        <ProjectImage
+          src={project.image}
+          width={0}
+          height={0}
+          sizes="100vw"
+          style={{ width: "100%", height: "auto" }}
```

Comment:
please don't inline styles. The goal is to use only one styling paradigma, to reduce complexity and enhance code maintainability.